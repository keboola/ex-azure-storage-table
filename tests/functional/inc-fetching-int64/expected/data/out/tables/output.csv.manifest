{
    "primary_key": [
        "id"
    ],
    "columns": [
        "id",
        "int64"
    ],
    "incremental": true
}
