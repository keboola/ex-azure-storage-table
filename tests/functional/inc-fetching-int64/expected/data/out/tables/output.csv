"1","10"
"20","20"
"300","100"
"4000","40"
"50000","30"
